### PR TITLE
added support section

### PR DESCRIPTION
**Pull request by ganeshrvel**

### CODE REVIEW

File: docs/catalog/sinks/PagerDuty.rst [hunk 1]
```diff
@@ -97,3 +97,19 @@ Example Output:
...
+Support
+----------------------
+
+Ask for help, or just say hi!
+
+1. `Slack <https://bit.ly/robusta-slack>`_ - **link only works on desktop**
+2. `GitHub Issues <https://github.com/robusta-dev/robusta/issues>`_
+
+
+**Commercial Support**
```
Comment (aantn) [inline docs/catalog/sinks/PagerDuty.rst:110]: Can we remove this part?

**Review by aantn — APPROVED** [review]

File: docs/catalog/sinks/PagerDuty.rst (hunk 1 above)
Comment (ganeshrvel) [reply docs/catalog/sinks/PagerDuty.rst:110]: Will `3.  Email: support@robusta.dev` do?

Comment (aantn) [reply docs/catalog/sinks/PagerDuty.rst:110]: Sure.

Comment (ganeshrvel) [reply docs/catalog/sinks/PagerDuty.rst]: @aantn done!

Comment (ganeshrvel) [reply docs/catalog/sinks/PagerDuty.rst]: If the changes look good, i will merge it.

Comment (aantn) [reply docs/catalog/sinks/PagerDuty.rst]: I pushed the button for you :)

Comment (ganeshrvel) [reply docs/catalog/sinks/PagerDuty.rst]: Thank you!🙌🙌😌

**Pull request merged**